apps/09todo-jquery/script.js

```diff
@@ -1,5 +1,16 @@
 'use strict';
 
 $(document).ready(function() {
-  // App logic goes here
+    // App logic goes here
+    //Using jQuery, put a submit event listener on the form.
+    $('form').submit(function(event) {
+        event.preventDefault();
+        var todoText = $(this).find('#todo').val();
+        $('#todo-list').append('<li><input type="checkbox">' + todoText + '<button class ="remove">remove</button></li>').sortable();
```

Comment:
You only need to call `sortable()` once the container `ul` list. Place that logic at the top of your app otherwise your content won't be sortable till a form submission is made.